src/main/scala/software/purpledragon/sbt/lock/DependencyLockPlugin.scala

```diff
@@ -30,6 +31,7 @@ object DependencyLockPlugin extends AutoPlugin {
     val dependencyLockFile = settingKey[File]("lockfile to generate")
     val dependencyLockWrite = taskKey[File]("write dependencies to lockfile")
     val dependencyLockRead = taskKey[Option[DependencyLockFile]]("read dependencies from lockfile")
+    val dependencyLockExclusion = settingKey[ModuleFilter]("lockfile module exclusion filter")
```

Comment:
Can you change this to:

```suggestion
    val dependencyLockModuleFilter = settingKey[ModuleFilter]("exclusion filter for dependencies")
```